scripts/run_docker_build.sh

```diff
@@ -35,8 +35,9 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-conda update --yes conda conda-build=1.20.0
+conda update --yes conda conda-build
 conda install --yes anaconda-client conda-build-all
+conda install --yes conda-build=1.20.0
```

Comment:
Let's just keep our eyes open for a new release with the bugfix. I guess that is bound to happen soon.